docs/Configurations.md

```diff
@@ -1,99 +1,86 @@
 # FCA: Configurations
 
-TODO: add intro
+This document gives the various configurations available for Firebolt Certification App (FCA). It provides insights into setting up and utiizing different environments, parameters, and app types to facilitate effective testing and certification of applications using firebolt technology. 
+
 
 ## Table of Contents
 
-- [Supported Targets](#supported-targets)
+- [Running with MFOS](#running-with-mfos)
 - [Connect to Mock Firebolt](#connect-to-mock-firebolt)
 - [Supported URL Parameters](#supported-url-parameters)
 - [App Types](#app-types)
 
-## Supported targets
-
-EXISTING CONTENTS(needs revision):
-> - MFOS - `https://github.com/rdkcentral/mock-firebolt`
+## Running with MFOS
 
 ### Background
 
-TODO: add details
+Mock Firebolt (MFOS) is a simulated environment that FCA can interact with.
 
 ### Setup
 
-TODO: add details
+- *MFOS*: Accessible at [Mock-Firebolt](https://github.com/rdkcentral/mock-firebolt)
+- Use Mock Firebolt's [usage documentation](https://github.com/rdkcentral/mock-firebolt#usage-local) to get set up. 
 
 ### Usage
 
-TODO: add details
+Utilize MFOS for testing in a controlled, simulated environment. MFOS can be ideal for early-stage development and testing.
```

Comment:
This is not sufficient instructions for users to know how to use MFOS with FCA.

Reply:
in the setup there is link to all MFOS documentation where setup info is all documented. 
also updated usage to this :
Utilize MFOS for testing in a controlled, simulated environment. MFOS can be ideal for early-stage development and testing.
Use MFOS if you don't have a device with firebolt running on it, or don't have a device at all.

Reply:
Right but that is setup instructions. Can we add an usage instructions on how to use MFOS with FCA. Think of an example of how we use MFOS to enhance testing.

Reply:
on FCA side only the url param is required, that's the only difference to run it against MFOS.
using MFOS with FCA has good documentation in [MFOS](https://github.com/rdkcentral/mock-firebolt?tab=readme-ov-file#usage-local)

Reply:
So asking a question here, we have our normal nightly pipelines and we have a mfos pipeline - what is the difference in the purpose they serve? How do they differ?
I see here: https://github.comcast.com/lightning-automation/thorTV-Tests/blob/development/ci/tasks/mfos_certification.sh#L253-L259 we do upload specific files? Why/what is that for?

cc// @parag-pv

Reply:
For MFOS, the yaml/json files are uploaded to override method responses. By default MF would return the mock values, but for any method that requires a custom value to be returned (for validation in TTT), we upload the overrides from MFOS call flow repo https://github.comcast.com/lightning-automation/MFOS_Call_Flow

This is more of a TTT/suite thing than FCA, we wouldn't need to mention that for FCA documentation since it's nothing specific to FCA.

Reply:
okay thanks for the info.